iXBRLViewerPlugin/viewer/src/js/tableExport.js

```diff
@@ -16,11 +16,26 @@ export class TableExport {
         $('table', iframe).each(function () {
             const table = $(this);
             if (table.find(".ixbrl-element").length > 0) {
-                table.css("position", "relative");
-                const exporter = new TableExport(table, reportSet);
-                $('<div class="ixbrl-table-handle"><span>Export table</span></div>')
-                    .appendTo(table)
-                    .click(() => exporter.exportTable());
+                const containsAbsolute = Array.from(this.querySelectorAll("*"))
+                    .reduce(
```

Comment:
Is there a reason to use this implementation of `reduce()` instead of `some()`?

Reply:
No good one :)
Fixed.